### PR TITLE
eager styletransfer post

### PR DESCRIPTION
**Pull request by jjallaire**

### CODE REVIEW

File: _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd [hunk 1]
```diff
@@ -0,0 +1,443 @@
+---
+title: "Neural style transfer with eager execution and Keras"
+description: |
+  Continuing our series on combining Keras with TensorFlow eager execution, we show how to implement neural style transfer in a straightforward way. Based on this easy-to-adapt example, you can easily perform style transfer on your own images. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: http://www.rstudio.com/
```
Comment (jjallaire) [inline _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd]: https://www.rstudio.com

File: _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+bibliography: bibliography.bib
+date: 09-09-2018
+categories:
+  - Neural style transfer
```
Comment (jjallaire) [inline _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd]: We are using more pithy categories right now. I'd say you could leave this one out.

File: _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+If you've read the chapter on neural style transfer from [Deep Learning with R](https://tensorflow.rstudio.com/learn/resources.html), you may recognize some of the code snippets that follow.
+However, there is an important difference: This post uses TensorFlow eager execution, allowing for an imperative way of coding that makes it easy to map concepts to code.
+Just like previous posts on eager execution on this blog, this is a port of a [Google Colaboratory notebook](https://colab.research.google.com/github/tensorflow/models/blob/master/research/nst_blogpost/4_Neural_Style_Transfer_with_Eager_Execution.ipynb) that performs the same task in Python.
+
+As usual, please make sure you have the required package versions installed. And no need to copy the snippets - you'll find the complete code among the [Keras examples](https://github.com/rstudio/keras/tree/master/vignettes/examples/eager-styletransfer.R).
+
+
+## Prerequisites
+
+The code in this post depends on the development versions of several of the TensorFlow R packages. You can install these packages as follows:
```
Comment (jjallaire) [inline _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd]: I think we can change this to CRAN assuming we don't add any new functionality based on reviewing the post.

File: _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+
+```{r, layout="l-body", eval=TRUE, echo=FALSE}
+knitr::include_graphics("images/The_Great_Wave_off_Kanagawa.jpg")
+```
+
+We create a wrapper that loads and preprocesses the input images for us.
+As we will be working with VGG19, a network that has been trained on ImageNet, we need to transform our input images in the same way that was used training it. Later, we'll apply the inverse transformation to our combination image before displaying it.
+
+```{r}
+load_and_process_image <- function(path) {
```
Comment (jjallaire) [inline _posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd]: how about `load_and_preprocess_image()` ?

**Comment by skeydan** [comment]
thank you, all changes pushed to that branch

**Pull request merged**